electrum/gui/qt/transaction_dialog.py

```diff
@@ -848,6 +860,8 @@ def update_fee_fields(self):
         fee = tx.get_fee()
 
         self.size_e.setAmount(size)
+        fiat_size = self.main_window.format_fiat_and_units(fee)
+        self.size_f.setAmount(fiat_size)
```

Comment:
please do not call this `size`... that word is used to mean the tx size in bytes everywhere else in this file.
it can be be `fiat_fee` or similar.